### ISSUE TITLE
App Installer Authentication Manager - Minimum Windows version ?

### ISSUE THREAD

**Issue by Styxxy** [MicrosoftDocs/msix-docs#250]
It is not specified in the docs, also information/documentation about this feature is very scarce .

Is there any minimum Windows version (or other product version) requirement to use this functionality? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8ff43162-0bdc-2c56-2a88-f8da9d274ba6
* Version Independent ID: 98f3077c-49cb-f48c-2b45-775e016b367d
* Content: [App Installer Authentication Manager - MSIX](https://docs.microsoft.com/en-us/windows/msix/app-installer/app-installer-authentication-manager--overview)
* Content Source: [msix-src/app-installer/app-installer-authentication-manager--overview.md](https://github.com/microsoftdocs/msix-docs/blob/master/msix-src/app-installer/app-installer-authentication-manager--overview.md)
* Product: **msix**
* GitHub Login: @dianmsft
* Microsoft Alias: **diahar**

**Comment by tcrevier** [comment]
> It is not specified in the docs, also information/documentation about this feature is very scarce .
> 
> Is there any minimum Windows version (or other product version) requirement to use this functionality?
> 

I agree!  I am trying to use this feature to deploy LOB apps through an Azure web site with Azure AD authentication.  There are no examples of how to use the new feature.  I would assume this would be a pretty standard method for deploying LOB apps, I am surprised it is not documented and appears not to work.  Disappointing.

**Comment by Marv51** [comment]
With the disabling of the ms-appinstaller URL scheme this feature is completely non functional, correct?
